.github/workflows/run-prerelease.yml

```diff
@@ -0,0 +1,72 @@
+name: Release Activiti Cloud
```

Comment:
If this is specific to Activiti, it should be reflected in the workflow name

Reply:
This is not a reusable workflow, but I should fix the field `name` that's not reflecting the prerelease.

Reply:
I updated the name so it doesn't conflict with `run-release.yml`.